pkg/testdata/check1.sarif

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
add exception for license

Reply:
@naveensrinivasan I'm going to need to a few more exceptions for https://github.com/ossf/scorecard/blob/main/.github/workflows/main.yml#L70
Can I just modify as `"**/script-empty.sh,pkg/testsdata/*"`?
All my files under `pkg/testdata/` need to be excluded.